manager2/src/app/user/user.component.ts

```diff
@@ -352,13 +335,13 @@ export class UserComponent implements OnInit {
             }
         }
         if (!found) {
-          this.groups.push({name: this.user.group, new: true});
-          this.missing_group = this.user.group;
+          this.groups.push(this.groupService.mapToGroup({ name: this.user.group, new: true }));
+          this.group_exists = false;
```

Comment:
I 'think' the reason I used `this.missing_group = this.user.group;` and then an `if user.group == this.missing_group` was that if you switch the selected group, you want the 'create group' button to go away.

By setting a static true / false variable, the button will always stay even if I select an existing group

Reply:
You're right. Modification reverted.